### PR TITLE
Add another method for configuration reload

### PR DESCRIPTION
**Pull request by szandala**
I've added missing method for configuration reload

### Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [ ] Link to relevant issues in GitHub or in [Jenkins JIRA](https://issues.jenkins-ci.org)
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Did you provide a test-case? That demonstrates feature works or fixes the issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,10 +338,10 @@ secrets:
 
 We don't support installing plugins with JCasC you need to use something else for this,
 
-Dockers users can use:  
+Dockers users can use:
```
Comment (casz) [inline README.md]: On markdown please do no trim double white spaces. Double white spaces and newline creates a line break
https://gist.github.com/shaunlebron/746476e6e7a4d698b373
```suggestion
Dockers users can use:  
```

File: README.md [hunk 2]
```diff
@@ -338,10 +338,10 @@ secrets:
...
 [https://github.com/jenkinsci/docker/#preinstalling-plugins](https://github.com/jenkinsci/docker/#preinstalling-plugins)
 
-Kubernetes users:  
+Kubernetes users:
```
Comment (casz) [inline README.md]: Ditto
```suggestion
Kubernetes users:  
```

File: README.md [hunk 3]
```diff
@@ -372,7 +378,7 @@ You have the following option to trigger a configuration reload:
   Allows to resolve secrets from AWS' SSM secrets
 - [hashicorp-vault-plugin](https://github.com/jenkinsci/hashicorp-vault-plugin)
   Allows to resolve secrets from Hashicorp vault
-  
+
```
Comment (casz) [inline README.md]:
```suggestion
```

**Comment by oleg-nenashev** [comment]
@szandala hi, are you still interested?

**Comment by szandala** [comment]
Overall, yes I am.
I had a flu for almost a week and I have limited my duties :p
I will check the complains

**Comment by szandala** [comment]
@casz for adding newline cannot we use slash?

**Comment by casz** [comment]
I was not aware of `\` for newlines 😱 What MAGIC

**Comment by szandala** [comment]
Now what then?

**Comment by szandala** [comment]
Btw. can I add label:
Hacktoberfest ?
It was no planned, but since I can get a T-shirt :p

**Comment by timja** [comment]
> Btw. can I add label:
> Hacktoberfest ?
> It was no planned, but since I can get a T-shirt :p

I don't think you need the label? but have added anyway

**Comment by oleg-nenashev** [comment]
@szandala Hacktoberfest won't work retrospectively for old PRs IIRC, but let's see

**Comment by szandala** [comment]
> @szandala Hacktoberfest won't work retrospectively for old PRs IIRC, but let's see

yeah, they told me that now...

Who can merge the PR anyway?

**Comment by timja** [comment]
> > @szandala Hacktoberfest won't work retrospectively for old PRs IIRC, but let's see
> 
> yeah, they told me that now...
> 
> Who can merge the PR anyway?

anyone who gives the green ✅, was just giving time for others to review.

**Pull request merged**